internal/resource/config/oauth/authserversettings/scopes/exclusivescope/oauth_auth_server_settings_scopes_exclusive_scope_resource.go

```diff
@@ -19,33 +19,33 @@ import (
 
 // Ensure the implementation satisfies the expected interfaces.
 var (
-	_ resource.Resource                = &oauthAuthServerSettingsScopesExclusiveScopesResource{}
-	_ resource.ResourceWithConfigure   = &oauthAuthServerSettingsScopesExclusiveScopesResource{}
-	_ resource.ResourceWithImportState = &oauthAuthServerSettingsScopesExclusiveScopesResource{}
+	_ resource.Resource                = &oauthAuthServerSettingsScopesExclusiveScopeResource{}
+	_ resource.ResourceWithConfigure   = &oauthAuthServerSettingsScopesExclusiveScopeResource{}
+	_ resource.ResourceWithImportState = &oauthAuthServerSettingsScopesExclusiveScopeResource{}
 )
 
-// OauthAuthServerSettingsScopesExclusiveScopesResource is a helper function to simplify the provider implementation.
-func OauthAuthServerSettingsScopesExclusiveScopesResource() resource.Resource {
-	return &oauthAuthServerSettingsScopesExclusiveScopesResource{}
+// OauthAuthServerSettingsScopesExclusiveScopeResource is a helper function to simplify the provider implementation.
+func OauthAuthServerSettingsScopesExclusiveScopeResource() resource.Resource {
+	return &oauthAuthServerSettingsScopesExclusiveScopeResource{}
 }
 
-// oauthAuthServerSettingsScopesExclusiveScopesResource is the resource implementation.
-type oauthAuthServerSettingsScopesExclusiveScopesResource struct {
+// oauthAuthServerSettingsScopesExclusiveScopeResource is the resource implementation.
+type oauthAuthServerSettingsScopesExclusiveScopeResource struct {
 	providerConfig internaltypes.ProviderConfiguration
 	apiClient      *client.APIClient
 }
 
-type oauthAuthServerSettingsScopesExclusiveScopesResourceModel struct {
+type oauthAuthServerSettingsScopesExclusiveScopeResourceModel struct {
 	Id          types.String `tfsdk:"id"`
 	Name        types.String `tfsdk:"name"`
 	Description types.String `tfsdk:"description"`
 	Dynamic     types.Bool   `tfsdk:"dynamic"`
 }
 
 // GetSchema defines the schema for the resource.
-func (r *oauthAuthServerSettingsScopesExclusiveScopesResource) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
+func (r *oauthAuthServerSettingsScopesExclusiveScopeResource) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
 	schema := schema.Schema{
-		Description: "Manages a OauthAuthServerSettingsScopesExclusiveScopes.",
+		Description: "Manages a OauthAuthServerSettingsScopesExclusiveScope.",
```

Comment:
Same thing here => Looks like this needs to be something like "Manages an Oauth Auth Server Settings Scopes Exclusive Scope"